coroutines-java/src/main/java/com/baeldung/App.java

```diff
@@ -0,0 +1,11 @@
+package com.baeldung;
```

Comment:
Let's move this code to an "introduction" package.  So the package should be `com.baeldung.introduction`.